connect_labs/labs/indicators/tests/test_resolve.py

```
@@ -0,0 +1,655 @@
+            boundary_id="AGO-2-1",
+            geometry=box(1, 1, 2, 2),
+            source=AdminBoundary.Source.GEOBOUNDARIES,
+        )
+        return country, province, district
+
+    def test_a_district_reaches_its_province_once_linked(self):
+        from django.core.management import call_command
+
+        from connect_labs.labs.indicators.resolve import ancestors
+
+        country, province, district = self._nested()
```

Ruff (0.16.2)

[warning] Unpacked variable `country` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)